modules/core/shared/src/main/scala/lucuma/core/math/SignalToNoise.scala

```diff
@@ -112,6 +132,10 @@ object SignalToNoise {
   given Order[SignalToNoise] =
     Order.fromLessThan(_ < _)
 
+  given AdditiveCommutativeSemigroup[SignalToNoise] = new AdditiveCommutativeSemigroup[SignalToNoise] {
+    def plus(x: SignalToNoise, y: SignalToNoise): SignalToNoise = (x + y)
```

Comment:
Because I chose to make the underlying type simple (a `long`) this compiles but produces an illegal value sometimes?   What is `Max` + `Max` for example?

Maybe I should have used the refined type range for the representation.

Reply:
I wasn't aware that we needed to add them, but if we do really add would be `(SignalToNoise, SignalToNoise) => Option[SignalToNoise]`?

Reply:
because of the range limits?

Reply:
Right, if we can end up with a `SignalToNoise` value that is bigger than `Max` the database would reject it when it processes inputs.  A `reverseGet` result passed to `getOption` on one of the optics would produce `None`, etc.

Reply:
Ok, let's rather remove the instance, we can convert to numbers and do operations that way